numerals/scripts/initializedb.py

```diff
@@ -189,56 +197,58 @@ def main(args):
                 )
         DBSession.flush()
 
+        _n = ds.column_names
```

Comment:
I think, `ns` for namespace would be a bit more transparent than `_n` - with no real justification for the `_`.